sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -10,15 +10,17 @@ import { AbortSignalLike } from '@azure/abort-controller';
 export type CancelOnProgress = () => void;
 
 // @public (undocumented)
-export abstract class Poller<TState, TResult> implements PollerLike<TState, TResult> {
-    constructor(operation: PollOperation<TState, TResult>);
+export abstract class Poller<TState, TResult, TPossibleResponse = unknown> implements PollerLike<TState, TResult, TPossibleResponse> {
+    constructor(operation: PollOperation<TState, TResult, TPossibleResponse>);
     // (undocumented)
     cancelOperation(options?: {
         abortSignal?: AbortSignal;
     }): Promise<void>;
     // (undocumented)
     protected abstract delay(): Promise<void>;
     // (undocumented)
+    getLastResponse(): TPossibleResponse | Error | undefined;
```

Comment:
maybe `getLastResponseOrError()`?  There's already a `getResult()` so `getLastResult()` could be confusing.

Reply:
`getResult` and `getLastResponse` seem ok to me, from a confusion perspective, but definitely open to other ideas!